narwhals_daft/expr.py

```diff
@@ -807,7 +807,6 @@ def str(self) -> ExprStringNamespace:
     last = not_implemented()
 
     # namespaces
-    dt = not_implemented()  # pyright: ignore[reportAssignmentType]
```

Comment:
In this file you need to add `@property` for `dt` similar to the PR that I linked